components/form/FormList.tsx

```diff
@@ -56,22 +58,26 @@ const FormList: React.FC<FormListProps> = ({
     }),
     [prefixCls],
   );
+  const { name: formName } = React.useContext(FormContext);
+  const fieldId = getFieldId(getNamePath(props.name), formName);
 
   return (
-    <List {...props}>
-      {(fields, operation, meta) => (
-        <FormItemPrefixContext.Provider value={contextValue}>
-          {children(
-            fields.map((field) => ({ ...field, fieldKey: field.key })),
-            operation,
-            {
-              errors: meta.errors,
-              warnings: meta.warnings,
-            },
-          )}
-        </FormItemPrefixContext.Provider>
-      )}
-    </List>
+    <div id={fieldId}>
```

Comment:
It's a break change since user may use List as inline elements

Reply:
ping ~